hyperdb/schema/schema.json

```diff
@@ -21,6 +21,31 @@
         }
       ]
     },
+    {
+      "name": "bundle",
+      "namespace": "pear",
+      "compact": false,
+      "flagsPosition": 1,
+      "fields": [
+        {
+          "name": "key",
```

Comment:
lets call this link, because in the codebase the concept of a key|path is called a link